src/Components/CreateImageWizard/steps/imageName.js

```diff
@@ -43,5 +51,23 @@ export default {
         },
       ],
     },
+    {
+      component: componentTypes.TEXT_FIELD,
```

Comment:
Text area might be better than text input in this case as it provides more space for the text and wraps based on the width of the modal - https://www.data-driven-forms.org/provided-mappers/textarea?mapper=pf4

`component: componentTypes.TEXTAREA`

Also the character count does not seem to work for me :(

![Screenshot from 2023-07-25 12-52-33](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/bab6b13e-a0a1-4ab2-89cb-93461b6302f4)

Reply:
changed to TEXTAREA